cmd/release-controller/types.go

```diff
@@ -226,6 +226,11 @@ type ReleaseVerification struct {
 	MaxRetries int `json:"maxRetries,omitempty"`
 	// AnalysisJobCount Number of asynchronous jobs to execute for release analysis.
 	AnalysisJobCount int `json:"analysisJobCount,omitempty"`
+	// AggregateProwJob requires that the named ProwJob from the prow config pass before the
+	// release is accepted. The job is run only one time and if it fails the release
+	// is rejected.
+	// Defaults to "release-openshift-release-analysis-aggregator" if not specified.
+	AggregateProwJob *ProwJobVerification `json:"aggregateProwJob,omitempty"`
```

Comment:
how about a new type for AggregatedProwJobVerification

```go


type ReleaseVerification struct{
  // stuff
  AggregateProwJob *AggregatedProwJobVerification
}

type AggregatedProwJobVerification struct{
  ProwJob ProwJobVerification 
  AnalysisJobCount int
}

```

Then once we fix the config, you can remove the ReleaseVerification.AggregateProwJob

Reply:
Done!